inst/include/Rcpp/platform/compiler.h

```diff
@@ -56,6 +43,13 @@
     #if GCC_VERSION >= 40600
         #define IS_GCC_460_OR_LATER
     #endif
+    // Inform users of dated compiler (e.g. <= RHEL6 )
```

Comment:
Throw it out the window... The window... We'll throw it out the window... Err, I'm removing it. 

Do we still want to proceed in throwing a warning via: `configure.ac`? @eddelbuettel @jjallaire